jest.config.js

```diff
@@ -3,16 +3,19 @@ module.exports = {
     {
       displayName: "Browser tests",
       testEnvironment: "jsdom",
+      setupFilesAfterEnv: ["jest-extended"],
     },
     {
       displayName: "Node tests",
       testEnvironment: "node",
+      setupFilesAfterEnv: ["jest-extended"],
     },
     {
       displayName: "ESLint",
       runner: "jest-runner-eslint",
       testMatch: ["**/*.js"],
       testPathIgnorePatterns: ["/dist/", "/node_modules/"],
+      setupFilesAfterEnv: ["jest-extended"],
```

Comment:
I'm guessing that this isn't required to keep the ESLint checks running?

Reply:
Nope, removed.